openscm/definitions/openscm_units.txt

```diff
@@ -0,0 +1,66 @@
+# syntax is either:
+# for variables which relate to existing variables in the registry
+# name = relation to unit already in registry = other names
+# for variables which should define a new base unit
+# name = [long_name] = other names
+
+a = 1 * year = annum = yr
+h = hour
+# since kt is used for "knot" in the defaults:
+kt = 1000 * t
+
```

Comment:
I just found that `°C` and `°F` are note defined (but `°` is converted to `degree`):

```suggestion
degreeC = degC
degreeF = degF
```
although @rgieseke opposes ridiculous units such as `°F` ;) rightly!